Expecto.Sample/Expecto.Sample.fs

```diff
@@ -36,6 +36,70 @@ let printingFsCheck =
       maxTest = 128
   }
 
+module ColorizationTests =
+  type Person = {
+    Name: string
+    Age: int
+  }
+  type Weather = {
+    Type: string
+    Precipitation: string
+  }
+  type SillyTestType = {
+    A: int
+    Nom: string
+    Weather: Weather
+    Tweets: int
+  }
+
+  let colorizationTests =
+    testList "Colorization tests. These should all fail to diplay the diff" [
+      test "Testing diff colorization" {
+        let personA = {Name = "Kesam"; Age = 30}
```

Comment:
Can we also have a test for a huge object where the fields differ a lot?

Reply:
Absolutely. I just used tiny differences because those are the ones I'm usually struggling with spotting when the object is big and the diff colorization fails me.

I'll add a bigger object where many of the fields are wrong? Do you have an example in mind? Like {Address, Name, City, SecurityCard={Number, Expiration, CCV}, Pet=[{PetName=, PetSpecie}...]}  ish with many fields differentiating?

Reply:
No, it's enough if a field deep into a list of items has both long values in its fields and one of those fields differ. What I'm trying to trigger is that `%A` formats newlines if it gets to large fields.

Reply:
Ah yeah i thought so. Imo it works satisfactory now, but perhaps you want to checkout the branch and play with a diff like that yourself?

Reply:
Ah yeah i thought so. Imo it works satisfactory now, but perhaps you want to checkout the branch and play with a diff like that yourself?